### ISSUE TITLE
Make plugin compatible with QGIS processing

### ISSUE THREAD

**Issue by JakobMiksch** [NINAnor/NOFAInsert#57]
We could use some functionality of the plugin and make  processing tools out of it. For example downloading data from NOFA. Such a processing tool could then further be used for creating some workflows with QGIS.

http://docs.qgis.org/testing/en/docs/pyqgis_developer_cookbook/processing.html

**Comment by ninsbl** [comment]
That is actually a brilliant idea for the future!
That would also allow to expose modelling functionality via WPS using [wps4server](https://github.com/3liz/qgis-wps4server)!

**Comment by mdsnor** [comment]
Yes, that would be a great enhancement. It would be adding yet a new functionality beyond data insertion and data visualisation, and would potentially improve the user experience of the NOFA  db. Should be carefully designed, and worth creating a milestone for it.
